packages/components/src/dropdown-menu/style.scss

```diff
@@ -106,4 +106,13 @@
 			padding-left: 0.5rem;
 		}
 	}
+
+	.components-dropdown-menu__menu-item,
+	.components-menu-item,
```

Comment:
Is `components-menu-item` included for backwards compatibility? I can't find it being applied anywhere in the code.

Reply:
Thanks for the review!

Good point. I added it because it's defined here along with `.components-dropdown-menu__menu-item`:
https://github.com/WordPress/gutenberg/blob/c4b3c3c73745136c18343d06579b7cfe0abc272e/packages/components/src/dropdown-menu/style.scss#L52

But maybe it's legacy. I've removed it from the code that this PR touches.